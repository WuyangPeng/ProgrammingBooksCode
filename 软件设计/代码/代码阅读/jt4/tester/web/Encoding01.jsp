
<%@ page contentType="text/html;charset=SJIS" %>

<h2>Test Character</h2>
<h2><%="\u9b5a"%> Unicode = 9b5a</hr>
