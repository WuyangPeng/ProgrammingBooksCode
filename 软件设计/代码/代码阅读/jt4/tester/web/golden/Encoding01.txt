


<h2>Test Character</h2>
<h2>‹› Unicode = 9b5a</hr>
